validator/sawtooth_validator/execution/processor_handlers.py

```diff
@@ -44,6 +44,12 @@ def handle(self, connection_id, message_content):
         else:
             max_occupancy = request.max_occupancy
 
+        if request.request_header_style == \
+                processor_pb2.TpRegisterRequest.STYLE_UNSET:
+            header_style = processor_pb2.TpRegisterRequest.EXPANDED
```

Comment:
Suggest putting a comment here that this is the default for backwards-compatibility reasons.

Reply:
Done